### DISCUSSION TITLE
--credentials syntex ?

### DISCUSSION THREAD

**Discussion by meanwhenhungry** [grahampugh/erase-install#446] [Q&A] [answered]
On v31 and would like to pass the base 64 credentials that I've created with setcredentials.sh

Tried
--credentials mybase64 --i-know-the-security-risks-and-i-still-want-to-supply-a-password-in-plain-text
--credentials "mybase64" --i-know-the-security-risks-and-i-still-want-to-supply-a-password-in-plain-text
--credentials "username:pw" --i-know-the-security-risks-and-i-still-want-to-supply-a-password-in-plain-text
--credentials username:pw --i-know-the-security-risks-and-i-still-want-to-supply-a-password-in-plain-text

what am I doing wrong?

**Comment by grahampugh** [comment, marked as answer]
There's a mistake in the documentation somewhere. Use --very-insecure-mode (instead of --i-know-the-security-risks-and-i-still-want-to-supply-a-password-in-plain-text).

**Comment by meanwhenhungry** [comment]
Thanks for the reply and for making the best and long due overhaul for Mac updates.

The —i-know is listed from —help.
